docs/pages/blog/mui-core-v5.md

```diff
@@ -0,0 +1,748 @@
+- We have updated the supported browsers.
+  - IE: **partial**. We have kept the logic added in the past to support IE 11,
+    however, we have stopped actively working on it. We can't guarantee that it works correctly. It's discontinued.
+  - Edge: from 14 to **91**. The minimum version based on Chromium.
+  - Firefox: from 52 to **78**.
+  - Chrome: from 49 to **90**. We have [assumed](https://developers.google.com/search/blog/2019/05/the-new-evergreen-googlebot) that Googlebot is always using the latest version of Chrome.
+  - Safari: from 10 to **12.5**
+
+These changes have allowed us to save [6 kB gzipped](https://github.com/mui-org/material-ui/pull/22814#issuecomment-700995216) on the `@mui/material` package.
+
+## Design kits
+
+We now have design kits for the MUI components available in the three main design tools out there: [Figma](https://material-ui.com/store/items/figma-react/), [Sketch](https://material-ui.com/store/items/sketch-react/), and [Adobe XD](https://material-ui.com/store/items/adobe-xd-react/). If you work together with a designer or team of designers, having them use the design kits makes for easier communication, as you'll be speaking the same language around the components. It'll also help designers to understand how the library is structured, especially theming features and each component prop, enabling much faster development.
+
+<a href="https://mui.com/store/#design"><img loading="lazy" src="/static/blog/mui-core-v5/design-kits.png" alt="Screenshot of buttons in the Figma design kit" style="width: 700px; margin-bottom: 16px;" /></a>
+
+<p class="blog-description">The design kits</p>
+
+Figma is the first design kit that benefits from the v5 update, and we're more excited than ever to get the kits into designers' hands! We understand that designers are generally more creative and productive when their physical and digital work environment is looking fresh — that is why our kits are now cleaner in every aspect, leveraging naming, easy component-showcasing, and consistency.
+
+Some major changes to the components:
+
+- We’ve added the new colors that v5 introduces (error, warning, info & success)
+- All new components are covered
+- States such as hover have been added for all relevant components
+
+## What's next?
+
+Looking back, we are glad to say that, together, we have accomplished most of the objectives we defined in the [v4 release blog post](https://mui.com/blog/material-ui-v4-is-out/#whats-next).
+
+We are also proud to share that the monetization initiatives we have started right after the release of v4 are working, e.g. [the store](/blog/material-ui-v4-is-out/#premium-themes-store-✨). We have incorporated a company two years ago, grown between x2-3 a year since, and are now over 10 people at the company with a positive cashflow 🚀. We are on track to reach 16 people by December 2021.
```

Comment:
```suggestion
We are also proud to share that the monetization initiatives we have started right after the release of v4 are working, e.g. [the store](/blog/material-ui-v4-is-out/#premium-themes-store-✨). We have incorporated the company two years ago, grown between x2-3 a year since, and we're now over 10 people with a positive cashflow 🚀. We are on track to reach 16 people by December 2021.
```